Python/bytecodes.c

```diff
@@ -3743,13 +3743,12 @@ dummy_func(
             return frame;
         }
 
-        op(INSERT, (--)) {
+        op(INSERT, (stuff[oparg], top -- top, stuff[oparg])) {
             // Inserts TOS at position specified by oparg
             PyObject *tos = TOP();
             for (int i = 1; i < oparg + 1; i++) {
                 stack_pointer[i] = stack_pointer[i - 1];
```

Comment:
Hm, doesn't this repeat the first stack element over and over? Maybe `memmove()` would do what you want? (It is supposed to be good at overlaps, unlike `memcpy()`.)

Reply:
Woops yeah thanks for catching this!